debian/postinst

```diff
@@ -115,10 +135,27 @@ EOF
     fi
 }
 
+
+# If held fips packages exist, we are on a FIPS PRO machine with FIPS enabled
+mark_reboot_for_fips_pro() {
+    FIPS_HOLDS=$(apt-mark showholds | egrep -q 'fips|libssl1|openssh-client|openssh-server|linux-fips|openssl|strongswan')
```

Comment:
Just to be sure here, if we don't have any of those holds, would that command return `1` ? I am just afraid if that call could fail postinst, since we are using `set -e`

Reply:
yeah I think that set -e is causing failures, I would want to be loud and problematic during pkg install time if we do have unexpected failures, otherwise we might not get a bug about it and we'd have very problematic behavior in a non working postinst. The "good news" is that my branch broke postinst multiple times and did generate apt cmdline errors as a result which broke our integration tests. I think we should actually have an explicity behave step that validates during our base image build that ua-tools did install properly without error so we fail loudly during testing